src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -66,6 +65,9 @@ public Command parseCommand(String userInput) throws ParseException {
         case Command.COMMAND_GROUP_CAL:
             return new CalCommandParser().parse(arguments);
 
+        case Command.COMMAND_WORD_TASK:
```

Comment:
I think should use COMMAND _GROUP _TASK.

Reply:
Resolved. But there is still inconsistency between GROUP and WORD for our constants. Not sure how are they different?
![image](https://user-images.githubusercontent.com/43085789/76822400-d4c90080-684b-11ea-89f0-30dd9e2345e1.png)

Reply:
GROUP is for the "big" general ones: mod, task, cal, cmd, class, etc.
Help should probably be turned into GROUP too.
Might extend help later

Reply:
And WORD is for the specific command action: list, view, find, etc.
Not the best word-choice, but hey it works so far